tests/pytests/test_index_oom.py

```diff
@@ -11,6 +11,9 @@
 
 @skip(cluster=True)
 def test_stop_background_indexing_on_low_mem(env):
+    # Change the memory limit to 80% so it can be tested without colliding with redis memory limit
```

Comment:
What do you mean by "not colliding with redis memory limit"?
Also, do we have any tests with the new default (100%)? If not, I think we should have a basic one